week-2-practise/level-2/4-slice.js

```diff
@@ -5,7 +5,11 @@
 // - remove the item at the specified index
 
 function remove(arr, index) {
-  return; // complete this statement
+  if (typeof arr[arr.length - 1] == "string") {
```

Comment:
You can write this by using only splice. The first method takes the index where to start removing and the second argument how many items to remove. e.g. `splice(index, 1)`